src/Tools/BuildBoss/BuildBoss.csproj

```diff
@@ -7,6 +7,7 @@
     <SignAssembly>false</SignAssembly>
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <IsShipping>false</IsShipping>
+    <ApplicationManifest>$(RepositoryEngineeringDir)config\app.manifest</ApplicationManifest>
```

Comment:
We should update BuildBoss to ensure every project with OutputType of EXE also has an ApplicationManifest entry.

Reply:
Sounds good

Reply:
Can we instead set `ApplicationManifest` property in `Imports.targets` if the project sets `OutputType` is exe or winexe and the target is .NET Framework? I don't think the app manifest should be set when targeting .NET Core.

Reply:
I'm thinking this might actually be a good default in Arcade SDK -- if the project does not specify `ApplicationManifest` (or even in .NET Core SDK).